app/e2e/packetGroupPage.demo.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { Locator } from "@playwright/test";
+import { test, expect, TAG_DEMO_PACKETS } from "./tagCheckFixture";
+import { getInstanceRelativePath } from "./utils";
+
+// Tests which are only run against localhost, where we can assume we have the demo dataset packets
+test.describe("Demo packet group page", { tag: TAG_DEMO_PACKETS }, () => {
+  const packetId = "20240729-154652-95a6c08c";
+  let rows: Locator;
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./parameters");
+    const table = await page.getByRole("table");
+    await expect(table).toBeVisible();
+    rows = await table.getByRole("row");
+  });
+
+  // we can't assume new packets won't have been created, but should at least have those from the demo set
+  test("can see packet rows", async ({ baseURL }) => {
+    await expect(await rows.count()).toBeGreaterThanOrEqual(3);
+    const oldestRow = await rows.last();
+    const firstCell = await oldestRow.getByRole("cell").first();
+    const link = await firstCell.getByRole("link");
+    await expect(link).toHaveText(packetId);
+    const expectedHref = getInstanceRelativePath(baseURL, `parameters/${packetId}`);
+    await expect(await link.getAttribute("href")).toBe(expectedHref);
+    // We expect servers we're testing on to have either GB or US locale.
```

Comment:
```suggestion
    // We expect browsers we're testing on to have either GB or US locale.
```

No? I'd be surprised if we did any date rendering server side. The JS should pick up the browser defaults.

Reply:
Actually you can set the locale in the playwright config, that should make this more reliable..